src/Symfony/Bundle/WebProfilerBundle/Csp/ContentSecurityPolicyHandler.php

```diff
@@ -98,6 +99,15 @@ public function updateResponseHeaders(Request $request, Response $response)
         return $nonces;
     }
 
+    /**
+     * Allows the evaluation of code via functions like eval().
+     * If enabled 'unsafe-eval' will be set in the Content-Security-Policy.
+     */
+    public function enableEvaluation()
+    {
+        $this->evaluationEnabled = true;
```

Comment:
this is never disabled again, so this won't work well for projects handling multiple master requests.

Reply:
Changed it to a setter which resets the property in the WebDebugToolbarListener next to the place where it is enabled via a callback. I liked that more than building some logic which resets itself magically

Reply:
Where is it disabled?